modules/offers/server/controllers/offers.server.controller.js

```diff
@@ -608,6 +608,21 @@ exports.list = function (req, res) {
     }
   }
 
+  // offer maxGuests filter
+  if (filters.hasObjectFilter('offer')) {
+    // Accept only valid values, ignore the rest
+    // @link https://lodash.com/docs/#filter
```

Comment:
Validation was missing here but no need to use lodash, can just ensure with `parseInt(value, 10)` and maybe `Math.min(10, value)` to avoid larger than 10?